packages-content-model/roosterjs-content-model-api/lib/publicApi/table/editTable.ts

```diff
@@ -34,14 +34,20 @@ import {
  * @param editor The editor instance
  * @param operation The table operation to apply
  */
-export default function editTable(editor: IStandaloneEditor, operation: TableOperation) {
+export default function editTable(
+    editor: IStandaloneEditor,
+    operation: TableOperation,
+    skipSnapshot?: boolean
+) {
     editor.focus();
 
     editor.formatContentModel(
-        model => {
+        (model, context) => {
             const [tableModel, path] = getFirstSelectedTable(model);
 
             if (tableModel) {
+                context.skipUndoSnapshot = skipSnapshot;
```

Comment:
any reason we need to skip undo snapshot here?

Reply:
The workflow is:
1. Click the table inserter
2. Internally the inserter selects a cell in the table and calls editTable after
3. editTable gets the table with a selected cell and does the operation

The problem is that editTable will create a snapshot with the selected cell, so between 1 and 2, a pre-selection snapshot is created and editTable is made to not create another one.